project_config.yml

```
@@ -0,0 +1,132 @@
+  - CGAS-Season       # Clinical Global Assessment Scale (CGAS) Season
+  - Physical-Season   # season during which the physical measurements were taken
+  - Fitness_Endurance-Season # Season for fitness recording
+  - FGC-Season        # Functional Games Characteristics (FGC) Season
+  - BIA-Season        # Bioelectrical Impedance Analysis (BIA) Season
+  - PAQ_A-Season      # Physical Activity Questionnaire (PAQ) A Season
+  - PAQ_C-Season      # Physical Activity Questionnaire (PAQ) C Season
+  - PCIAT-Season      # PCIAT-Season
+  - SDS-Season      # Sleep Disturbance Scale (SDS) Season  
+
+# Target column
+target: sii
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)